src/components/messageSpace/messageSpace.tsx

```diff
@@ -19,26 +19,47 @@ export interface MessageSpaceProps extends MessageContainerProps {
   scrollDownLabel?: string
 }
 
+function usePrevious(value: number) {
```

Comment:
Could this have a more descriptive name? It's a bit unclear what's happening here